h3/src/proto/headers.rs

```diff
@@ -77,6 +86,7 @@ impl Header {
         //# If the scheme does not have a mandatory authority component and none
         //# is provided in the request target, the request MUST NOT contain the
         //# :authority pseudo-header or Host header fields.
+        trace!("got headers {:#?}", self.fields);
```

Comment:
I would recommend against logging the fields, this has caused problems in hyper (and plenty of internal privacy reviews of any language/framework), it's too easy to accidentally log sensitive information.

Reply:
makes sense